nixos/modules/misc/mandoc.nix

```diff
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
```

Comment:
Why split this out into a different file `/misc/mandoc` instead of having both implementations in the same file? The latter feels a lot nicer to navigate.

Reply:
Especially since this file is only like 50 lines long

Reply:
I just tried it and it is very annoying work merging the files whilst also making the resulting file a harder to understand, so I would like to at least postpone this change. I added a note to `documentation.nix` where the actual implementations are defined, so hopefully nobody gets lost at least.